doc/source/operations/rabbitmq.rst

```diff
@@ -0,0 +1,90 @@
+===============================
+Rolling out changes to RabbitMQ
+===============================
+
+Prior to upgrading, some changes need to be rolled out for RabbitMQ.
```

Comment:
Could do with a bit more background here. e.g.

* what you get in kolla-ansible by default & potential problems with it
* what you get when the flag is set
* when we change the default (xena)

Then move onto how to roll it out, with conditions for different starting versions